packages/data-layer/src/queries.ts

```diff
@@ -1,14 +1,13 @@
 import { gql } from "graphql-request";
 
 /**
- * Get all the programs that a user is a part of
- * @param $alloVersion - The version of Allo
+ * Manager: Get all the programs that a user is a part of in allo v1
  * @param $address - The address of the user
  * @param $chainId - The network ID of the chain
  *
  * @returns The programs
  */
-export const getProgramsByUser = gql`
+export const getV1ProjectsByUser = gql`
```

Comment:
these two separate queries are exactly the same you could have a single query here that receives a tag to filter by, the caller can use program for v1 or allo-v2 for v2